server/src/test/resources/csars/yaml/valid/lamp-csar/scripts/myphpapp.php

```diff
@@ -0,0 +1,82 @@
+<?php
```

Comment:
I would move the file out of the scripts folder.

Reply:
already done